CHANGELOG.md

```diff
@@ -9,10 +9,45 @@ A [separate changelog is kept for rand_core](rand_core/CHANGELOG.md).
 You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.html) useful.
 
 
-## [0.7.0 Pre-release] - 2019-06-06
-Many changes (TODO)
-- Enable fork protection of ReseedingRng without `std`
-- Remove dependency on `winapi`
+## [0.7.0] - 2019-??-??
+
+### Fixes
+- Fix incorrect pointer usages revealed by Miri testing (#780, #781)
+- Fix (tiny!) bias in `Uniform` for 8- and 16-bit ints (#809)
+
+### Crate
+- Bumped MSRV (min supported Rust version) to 1.32.0
+- Updated to Rust Edition 2018  (#823, #824)
+- Removed dependence on `rand_xorshift`, `rand_isaac`, `rand_jitter` crates (#759, #765)
+- Remove dependency on `winapi` (#724)
+- Removed code already deprecated in version 0.6 (#757)
+- Many documentation changes
```

Comment:
I would maybe mention the removal of all `build.rs`, which improves build time.

Reply:
Done